docs/2.9.0/docs/query/pqs-user-guide.rst

```diff
@@ -1,466 +1,1741 @@
+   require any changes to the schema as the ledger evolves, to the
+   extent that this is possible. Note: discovering adding new templates
+   does currently require additional table partitions to be added.
+
+Design
+~~~~~~
+
+To facilitate these objectives, the following design approaches have
+been used:
+
+-  **Concurrent append-only writes**: ledger transactions are written
+   with significant parallelism without contention, ensuring that writes
+   can be high-throughput and be unconstrained by latency.
+-  **Bulk batching**: using
+   `COPY <https://www.postgresql.org/docs/current/sql-copy.html>`__ (not
+   `INSERT <https://www.postgresql.org/docs/current/sql-insert.html>`__)
+   to deliver large batches of data efficiently.
+-  **Offset indexed**: all data is appropriately indexed by offset to
+   provide efficient access to slice the result by offset.
+   `BRIN <https://www.postgresql.org/docs/current/brin-intro.html>`__
+   indexes are used to ensure contiguity of data that is often accessed
+   together.
+-  **Implicit offset**: readers can opt for queries with implicit
+   offset, meaning they can ignore the role of offset in their queries -
+   but still provide a stable view of the ledger data. Much like
+   PostgreSQL's MVCC provides consistency benefits without the user
+   needing to understand the underlying implementation, we seek to
+   provide a similar experience for readers of the ledger data.
+-  **Idempotent**: PQS is designed to be restarted at any time, and will
+   not impact the integrity of the data. This is achieved by using
+   offsets to ensure clean restarts, and ensuring that (other than the
+   database itself) PQS is stateless.
+-  **Watermarks**: a single thread maintains a watermark denoting the
+   most recent contigous transaction - representing the offset of the
+   ledger that is consistent and available. In addition, the watermark
+   processes the "archive" mutation on any archived contracts or
+   interface views, on a batched basis. This reintroduces data
+   consistency without needing readers to perform pathalogical table
+   scans to achieve this. This efficiently resolves the uncertainty
+   created by the parallel writes.
+-  **Schemaless content**: content defined in Daml templates uses the
+   JSONB datatype to store the data. This allows for a schemaless
+   approach, and is able to store any Daml model without needing to
+   change the schema - other than custom JSONB indexes.
+
+Resilience
 ==========
 
-The PQS is intended for continuous operation. Upon restart after an interruption, PQS determines the last consistent offset and continues incremental processing from that point onward. PQS terminates when encountering any error and leaves it up to the orchestration layer (such as Kubernetes) or the operator to determine the appropriate course of action.
+PQS operates as a long-running process, and faces many potential threats
```

Comment:
As written, this can be concerning to thin skinned customers.  Could I suggest:  "PQS is designed to operate as a long-running process which uses these principles to enhance availability:"